File: Consulta de Representación del conocimiento/Reprecentacion del conocimiento/Reprecentacion del conocimiento Java/src/reprecentacion/del/conocimiento/ReprecentacionDelConocimiento.java
package reprecentacion.del.conocimiento;

/**
 *
 * @author Daniel Aviles, DeltaTeam, DCCO-ESPE
 */
public class ReprecentacionDelConocimiento {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        /*En este ejemplo de la Representacion del conocimiento
        se usa una clase Animal para retener y posteriormente 
        mostrar(usar) la informacion sobre un animal*/
        Animal perro = new Animal("Perro", 3, "Max");
        perro.mostrarInformacion();
    }
    
}

File: Consulta de Representación del conocimiento/Reprecentacion del conocimiento/Reprecentacion del conocimiento.py
"""En este ejemplo de la Representacion del conocimiento
se usa una clase Animal para retener y posteriormente 
mostrar(usar) la informacion sobre un animal"""
class Animal:
    def __init__(self, especie, edad, nombre):
        self.especie = especie
        self.edad = edad
        self.nombre = nombre
    
    def mostrarInformacion(self):
        print("Especie:", self.especie)
        print("Edad:", self.edad, "años")
        print("Nombre:", self.nombre)

perro = Animal("Perro", 3, "Max")
perro.mostrarInformacion()
